NitroxClient/Communication/Packets/Processors/WeldActionProcessor.cs

```diff
@@ -0,0 +1,52 @@
+﻿using NitroxClient.Communication.Abstract;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    class WeldActionProcessor : ClientPacketProcessor<WeldAction>
```

Comment:
can this be made more generic for LiveMixing?  For instance, we may want a system were some knifes a fish and wants the same behavior (packet sent out for simulator to update health).

Reply:
We can do a generic LiveMixinChangeRequest. The idea behind a destinct WeldAction packet and processor was that for other actions (knifing a fish) it may not be sufficient to only change the health.
For example hitting with a knife: If we then want to also synchronize slashing sounds, only requesting health change will not cut it.